Fix CSP example explanation (#34286)
mdn/content: files/en-us/web/http/csp/index.md

### Example 5

A website administrator of a web mail site wants to allow HTML in email, as well as images loaded from anywhere, but JavaScript or other potentially dangerous content can only come from the same origin as the mail server.

```http
Content-Security-Policy: default-src 'self' *.example.com; img-src *
```

Note that this example doesn't specify a {{CSP("script-src")}}; with the example CSP,

> this site uses the setting specified by the {{CSP("default-src")}} directive.

Reviewer: What do you think about this?

```suggestion
so the {{CSP("default-src")}} directive will be used for JavaScript sources as a fallback.
```
